UPGRADE_GUIDE.md

```diff
@@ -2,22 +2,130 @@
 
 Use the following guide to assist in the upgrade process of the `easypost-java` library between major versions.
 
-* [Upgrading from 4.x to 5.0](#upgrading-from-4x-to-50)
+- [Upgrading from 5.x to 6.0](#upgrading-from-5x-to-60)
+- [Upgrading from 4.x to 5.0](#upgrading-from-4x-to-50)
+
+## Upgrading from 5.x to 6.0
+
+### 6.0 High Impact Changes
+
+- [Client Instance](#60-client-instance)
+- [Error Types](#60-error-types)
+- [Uniform empty response](#60-uniform-empty-response)
```

Comment:
We know it as uniform empty responses, but let's say what it is from the users perspective. Please fix the links too

```suggestion
- [Empty Response Function Return Types](#60-uniform-empty-response)
```